src/Display.css

```diff
@@ -0,0 +1,9 @@
+input {
+	font-family: 'Calc';
+	font-size: 1.4em;
+}
+
+@font-face {
+font-family: 'calc';
+src: url('calc-font.ttf');
+}
```

Comment:
Add newline to end of File. Hopefully your editor does this when saving a file.

If it doesn't, consider finding a way to make that happen.